tests/integration/components/sl-date-time-test.js

```diff
@@ -0,0 +1,204 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-date-time', 'Integration | Component | sl date-time', {
+    integration: true
+});
+
+test( 'Defaults applied correctly', function( assert ) {
+
+    this.render( hbs`
+        {{sl-date-time
+            timezone="America/Chicago"
+        }}
+    ` );
+
+    const defaultRendered = this.$( '>:first-child' ).text().trim();
+    const defaultRegEx = /^[a-zA-Z]+[,]\s[a-zA-Z]+\s\d{1,2}[a-z]{2}\s\d{4}[,]\s\d{1,2}[:]\d{2}\s(AM|PM)\s[A-Z]+$/;
+
+    assert.deepEqual(
+        defaultRegEx.test( defaultRendered ),
+        true,
+        'Default datetime string matches default pattern'
+    );
+
+    const datetimeAttr = this.$( '>:first-child' ).attr( 'datetime' );
+    const dataOriginalTitleAttr = this.$( '>:first-child' ).attr( 'data-original-title' );
+
+    assert.strictEqual(
+        datetimeAttr,
+        dataOriginalTitleAttr,
+        'Attributes datetime and data-original-title match'
+    );
+
+    assert.deepEqual(
+        /^\d{4}[-]\d{2}[-]\d{2}\s\d{1,2}[:]\d{2}\s[A-Z]+$/.test( this.$( '>:first-child' ).attr( 'datetime' ) ),
+        true,
+        'Attribute datetime matches ISO datetime format plus timezone code'
+    );
+
+    const element = this.$( '>:first-child' );
+    const data = element.data();
+    const tooltipData = data[ 'bs.tooltip' ];
+    const options = tooltipData.getOptions();
+
+    assert.strictEqual(
+        tooltipData.enabled,
+        true,
+        'tooltip is enabled'
+    );
+
+    assert.strictEqual(
+        tooltipData.getTitle(),
+        dataOriginalTitleAttr,
+        'Title text is set correctly'
+    );
+
+    assert.strictEqual(
+        options.trigger,
+        'hover focus',
+        'Default trigger is "hover focus"'
+    );
+});
+
+test( 'Relative values applied correctly', function( assert ) {
+
+    const pastDate = moment().subtract( 4321, 'minutes' ).toISOString();
```

Comment:
Use `window.moment` to be consistent with what we are doing in other tests and component code.